frontend/src/components/VMediaInfo/VCopyLicense.vue

```diff
@@ -39,6 +39,15 @@
       >
         <p>{{ getAttributionMarkup({ isPlaintext: true }) }}</p>
       </VLicenseTabPanel>
+      <VLicenseTabPanel
+        :tab="tabs[3]"
+        :media-id="media.id"
+        :media-type="media.frontendMediaType"
+      >
+        <!-- Ignore reason: the interpolated string cannot have any whitespace around it when inside <p>, else there will be unwanted whitespace -->
+        <!-- prettier-ignore -->
+        <p id="attribution-xml" class="break-all font-mono" dir="ltr">{{ getAttributionMarkup({ isXml: true }) }}</p>
```

Comment:
It might be good to use `pre` here, instead of `p`, so that the XML snippet retains its line breaks. Although, this approach with `p` matches the existing HTML version :shrug: Both are a difficult and annoying jumble to read, which might make sense for HTML (maybe more people copy/paste HTML without checking it) but for XML, it's definitely targeted at more technical users, so I think it could be formatted in presentation?

```suggestion
        <pre dir="ltr">{{ getAttributionMarkup({ isXml: true }) }}</pre>
```